src/unix/linux_like/linux/gnu/b64/powerpc64/mod.rs

```diff
@@ -190,6 +196,44 @@ s! {
         pub ss_flags: ::c_int,
         pub ss_size: ::size_t
     }
+
+    pub struct vscr_t {
+        if #[cfg(target_arch = "powerpc64")] {
+            __pad: [::c_uint; 3],
+            vscr_word: ::c_uint,
+        } else {
+            vscr_word: ::c_uint,
+            __pad: [::c_uint; 3],
+        }
+    }
+
+    pub struct vrregset_t {
+        pub vrregs: [[::c_uint; 4]; 32],
+        pub vscr: ::vscr_t,
+        pub vrsave: ::c_uint,
+        _pad: [::c_int; 3],
+    }
+
+    pub struct mcontext_t {
+        __glibc_reserved: [::c_ulong; 4],
+        pub signal: ::c_int,
+        __pad0: ::c_int,
+        pub handler: ::c_ulong,
+        pub oldmask: ::c_ulong,
+        pub regs: *mut pt_regs,
+        pub gp_regs: ::gregset_t,
+        pub fp_regs: ::fpregset_t,
+        pub v_regs: *mut vrregset_t,
+        pub vpmx_reserve: [::c_long; 69]
+    }
```

Comment:
glibc source: https://github.com/bminor/glibc/blob/master/sysdeps/unix/sysv/linux/powerpc/sys/ucontext.h#L120-L150

```c
typedef struct {
	unsigned long	__glibc_reserved[4];
	int		__ctx(signal);
	int		__pad0;
	unsigned long	__ctx(handler);
	unsigned long	__ctx(oldmask);
	struct __ctx(pt_regs)	*__ctx(regs);
	gregset_t	__ctx(gp_regs);
	fpregset_t	__ctx(fp_regs);
/*
 * To maintain compatibility with current implementations the sigcontext is
 * extended by appending a pointer (v_regs) to a quadword type (elf_vrreg_t)
 * followed by an unstructured (vmx_reserve) field of 69 doublewords.  This
 * allows the array of vector registers to be quadword aligned independent of
 * the alignment of the containing sigcontext or ucontext. It is the
 * responsibility of the code setting the sigcontext to set this pointer to
 * either NULL (if this processor does not support the VMX feature) or the
 * address of the first quadword within the allocated (vmx_reserve) area.
 *
 * The pointer (v_regs) of vector type (elf_vrreg_t) is essentially
 * an array of 34 quadword entries.  The entries with
 * indexes 0-31 contain the corresponding vector registers.  The entry with
 * index 32 contains the vscr as the last word (offset 12) within the
 * quadword.  This allows the vscr to be stored as either a quadword (since
 * it must be copied via a vector register to/from storage) or as a word.
 * The entry with index 33 contains the vrsave as the first word (offset 0)
 * within the quadword.
 */
	vrregset_t	*__ctx(v_regs);
	long		__ctx(vmx_reserve)[__NVRREG+__NVRREG+1];
} mcontext_t;
```